app/views/content_items/field_of_operation.html.erb

```diff
@@ -0,0 +1,68 @@
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= render "govuk_publishing_components/components/title", @content_item.title_and_context %>
+    </div>
+    <div class="govuk-grid-column-one-third govuk-!-padding-top-8 govuk-!-padding-left-0 govuk-!-padding-bottom-8">
+      <%= render "govuk_publishing_components/components/organisation_logo", {
+        organisation: {
+          name: sanitize("Ministry<br>of Defence"),
+          url: "/government/organisations/ministry-of-defence",
```

Comment:
Could we get this data from the content item's links rather than hard coding it?

Reply:
good shout, have amended